ui/src/app/views/concept-homepage/component.ts

```diff
@@ -238,12 +243,52 @@ export class ConceptHomepageComponent implements OnInit {
 
   selectConcept(selectedConcepts) {
     this.selectedConcept = selectedConcepts;
-    if (this.selectedConcept && this.selectedConcept.length > 0 ) {
-      this.conceptSelected = true;
+    const domainName = this.selectedDomain.domain;
+    if (selectedConcepts && selectedConcepts.length > 0 ) {
+      const filterConceptsCount = this.selectedConcept
```

Comment:
its the same value since  this.selectedConcept is assigned the value selectedConcepts(argument of this function). I will change the argument name to avoid confusion.

Reply:
I just meant that the property name should be renamed as plural since it is an array.